google/cloud/storage/examples/storage_object_samples.cc

```diff
@@ -259,7 +259,18 @@ void GetObjectMetadata(google::cloud::storage::Client client,
 
     std::cout << "The metadata for object " << object_metadata->name()
               << " in bucket " << object_metadata->bucket() << " is "
-              << *object_metadata << "\n";
+              << "\n CacheControl : " << object_metadata->cache_control()
```

Comment:
nit: if `CacheControl` goes into the next line then the previous `" is "` should be `" is:"` ?

Also, `CacheControl` is not the name of the field in the JSON API (where it is `cacheControl`), in the HTTP headers (`Cache-Control`), nor in the C++ API (`cache_control()`).  Seems sad to introduce a name that matches none of the other fields the application developer might be familiar with?